docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+Opmerking AK Essentie hiervan is dat objecten die nu in het RGBZ en daarmee in de ZRC voorkomen maar ook relevantie hebben in een andere context of domein, als aparte registratie neergezet worden om recht te doen aan de betekenis als zelfstandige objecttype.
+* Dit kunnen zijn (niet uitputtend):
+    * Kernregistratie Klanten en Contacten (voor initiators,  belanghebbenden, gemachtigden e.d.)
+    * Kernregistratie medewerkers (o.a. voor behandelaars)
+    * Kernregistratie besluiten (besluiten)
+* Documentregistratiecomponent (documenten cq. informatieobjecten)
+    * Zaaktyperegistratiecomponent (Zaaktype, Statustypen, Resultaattypen,..)
+    * Domein/objectregistraties (domein/objectgegevens; meerdere registraties: één per domein/proces)
+    * Meldingenregistratie (ontvangen aanvragen en meldingen)
+    * Basisregistraties (personen, bedrijven, panden, terreinen, percelen e.d.)
+    * Evt. andere kernregistraties (Zaakobjecten die nog niet genoemd zijn, maar wel behoren tot kerngegevens (meervoudig gebruikt))
+
+
+
+Figuur 2: ZGW in een gegevenslandschap
+
+## 1.4 Vraagstukken
+
+Bovenstaande plaat schetst het concept ZGW in een gegevenslandschap al heel aardig.
+
+Toch zijn er nog enkele openstaande vragen te beantwoorden. Voor een deel is dit al af te leiden uit de stippellijnen in bovenstaande plaat:
+
+### 1.4.1. Hoe verhoudt melding/verzoek zich tot een zaak?
+
+In bovenstaande figuur is de relatie tussen Melding/Aanvraag (Verzoek) en Zaak als een stippellijn getekend. Dit omdat Verzoek niet in het RGBZ voorkomt (zat wel in GFO Zaken) en we dus niet weten hoe Verzoek en Zaak zich tot elkaar verhouden. Essentie van de relatie lijkt te zijn dat een verzoek of melding een op zich zelf staand ‘iets’ is en behandeld wordt als zaak. Één verzoek kan leiden tot meerdere zaken. Meerdere Verzoeken kunnen ook leiden tot één Zaak?
+Documenten bij een Verzoek worden bij het Verzoek vastgelegd. Later ook rechtstreeks gerelateerd aan de zaak?
+
+### 1.4.2. Waar slaan we procesinformatie op?
+
+Het is nog onduidelijk waar we in een gegevenslandschap “Procesinformatie” willen opslaan. In bovenstande figuur is dit – samen met procesdefinitiegegevens - gepositioneerd in de Zaakafhandelcomponent (net als nu in de diverse Taaksystemen).
+Dit is echter (mogelijk) in strijd met het uitgangspunt in Common ground dat functionaliteit en gegevens van elkaar gescheiden moeten zijn. Procesgegevens zijn immers ook gegevens.
+De procesdefinitie is in veel gevallen zelfs impliciet (hard gecodeeerde flow tussen schermen); dat valt dan niet eens los te knippen.
```

Comment:
Kun je daar geen afspraken over maken dan?

Reply:
Ik begrijp het dilemma. De manier waarop leveranciers de zaakafhandeling inrichten verschilt, bijv. met workflow of met case management, of nog iets anders. Vraag is dan of je dat wel kunt scheiden van de ZAC.

Reply:
alles kan. zeker technisch, maar alles heeft ook zijn prijs..
je moet wel een interessant stukje markt voor leveranciers overlaten..

inmiddels ben ik zover dat ik de laatste zin van Hugo onderschrijf: is niet mogelijk.